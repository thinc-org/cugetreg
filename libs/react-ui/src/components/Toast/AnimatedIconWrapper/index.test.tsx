import { shallow } from 'enzyme'

describe('AnimatedIconWrapper', () => {
  const mockIcon = <div>icon</div>

  it('Should render correctly', async () => {
    const { AnimatedIconWrapper } = await import('.')
    const wrapper = shallow(<AnimatedIconWrapper>{mockIcon}</AnimatedIconWrapper>)

    expect(wrapper).toMatchSnapshot()
  })
})
