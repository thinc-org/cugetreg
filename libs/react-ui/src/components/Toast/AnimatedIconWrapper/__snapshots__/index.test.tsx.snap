// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnimatedIconWrapper Should render correctly 1`] = `
<Fragment>
  <Insertion
    cache={
      Object {
        "insert": [Function],
        "inserted": Object {},
        "key": "css",
        "nonce": undefined,
        "registered": Object {},
        "sheet": StyleSheet {
          "_insertTag": [Function],
          "before": null,
          "container": <head />,
          "ctr": 0,
          "insertionPoint": undefined,
          "isSpeedy": false,
          "key": "css",
          "nonce": undefined,
          "prepend": undefined,
          "tags": Array [],
        },
      }
    }
    isStringTag={true}
    serialized={
      Object {
        "map": undefined,
        "name": "j4d3wm",
        "next": Object {
          "name": "animation-1dnifoi",
          "next": undefined,
          "styles": "@keyframes animation-1dnifoi{
from {
  transform: scale(0.6);
  opacity: 0.4;
}
to {
  transform: scale(1);
  opacity: 1;
}}",
        },
        "styles": "
  display: flex;
  align-items: center;

  position: relative;
  transform: scale(0.6);
  opacity: 0.4;
  min-width: 20px;
  animation: animation-1dnifoi 0.3s 0.12s cubic-bezier(0.175, 0.885, 0.32, 1.275) forwards;
",
        "toString": [Function],
      }
    }
  />
  <div
    className="css-j4d3wm"
  >
    <div>
      icon
    </div>
  </div>
</Fragment>
`;
